sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -384,8 +509,46 @@ a compile error if `PropertyListT` does not contain a property with the
 Available only if `propertyT` is the property key class of a compile-time
 constant property.
 
+a|
+[source,c++]
+----
+template <typename T2, typename PropertyList2,
+          typename R = decltype(std::declval<T>() OP std::declval<T2>())>
+R operatorOP(const annotated_arg<T2, PropertyList2> &other) const;
+----
+|
+Defines the overloaded operators for types `annotated_arg<T, ...>` and
+`annotated_arg<T2, ...>`. Let `operatorOP` denote the operator used.
+The overloaded operator `operatorOP` utilizes `operatorOP(T2)` of type `T`
+and is available only if `operatorOP(T2)` is valid for objects of type `T`.
+The value and result is the same as the result of `operatorOP(T2)` applied to the
+objects of type `T` and `T2`.
```

Comment:
By my reading of this text, only member functions of T can be called, but that's not what the template checks for and it's not how you implemented it in the header files.  The text implies it was written like this:

```
  template <typename T2, typename PropertyList2,
    typename R = decltype(std::declval<T>() + std::declval<T2>())>
  R operator+(const annotated_arg<T2, PropertyList2> &other) const {
    return obj.operatorOP(other.obj);
  }
```

I think if you want it to match what you implemented you should instead say:

```suggestion
The overloaded operator `operatorOP` utilizes `operatorOP(T, T2)` and is available only if `operatorOP(T, T2)` is well formed.  
The value and result is the same as the result of `operatorOP(T, T2)` applied to the
objects of type `T` and `T2`.
```